webapp/public/js/domjudge.js

```diff
@@ -674,3 +674,32 @@ function pinScoreheader()
 		}
 	}
 }
+
+function removeMultipleUpdater(self, baseRoute)
```

Comment:
Instead of all these replaces, why not do the following:

* Save somewhere what the 'general' URL to the delete page is. Something like you already have as argument to this function but them maybe the whole url with a `_REPLACE_ME_` inside it.
* Keep track of an array of items to delete
* `var url = urlTemplate.replace('_REPLACE_ME_', items.join(','))`

Then we don't have all these strange replaces with ,, and ',